bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -15,7 +15,12 @@
 <article class="main">
 
   {# Navbar macro. Params: Page title, tweet text, Twitter URL, Facebook URL, Google+ URL #}
-  {{ smarton_navbar(_('Cybersecurity'), _('Look both ways, stay updated, and make sure your passwords are as buff as you are. Get smart on cybersecurity.') + hashtag, 'http://mzl.la/cybersecurity', 'https://www.mozilla.org/teach/smarton/security/', 'https://www.mozilla.org/teach/smarton/security/') }}
+  {# see bug 1188591, comment 10 for details regarding the need for the en-US conditional #}
+  {% if LANG == 'en-US' %}
```

Comment:
Not really, given that the page is already live (I sent an email yesterday to warn localizers about it). Unless we want to add this as an alternative string behind a tag.